### DISCUSSION TITLE
[1.18.1] Compatibility with Project MMO suggestion.

### DISCUSSION THREAD

**Discussion by JonCris** [vectorwing/FarmersDelight#425] [Ideas]
I would like this mod to be compatible with [Project MMO](https://www.curseforge.com/minecraft/mc-mods/project-mmo), I went to to their dev first but he says that it's not possible to include compatibility unless "farmers delight posts the slicing action to a forge event" "post their slice actions as crafting to the forge event bus".

Thank you o/

**Comment by vectorwing** [comment]
Hmm... Didn't think of utilizing an event there. Might be worth looking into.
Do they know which event I would need to send for it to work?

**Reply by Cromanelli2120** [reply]
Sorry to semi-resurrect this, but I believe the way to implement it would be to post a 'slice'/'craft' event to forge for the cutting board and skillet/cooking pot so the pmmo devs can use that event

**Reply by Caltinor** [reply]
@vectorwing Forge provides an `ItemCraftedEvent` that can be used for crafting.  as for right-click-harvesting, you would post a `BreakEvent` and `PlaceEvent` respectively.  It would be your decision whether or not use the cancellation status in your own logic.

-Caltinor, PMMO Dev

**Comment by Corwin00420** [comment]
I would like o second this request MC 1.18.2 Forge 40.1.52

**Comment by Caltinor** [comment]
Because it was mentioned,  adding an event for smelting/cooking is less easy.  In my comment above, the events referenced are built into Forge.  Forge does not, however, have an event for when things are smelted.  (maybe that will be my next PR).  Therefore to get any sort of cooking event, FD would need to post PMMO's `FurnaceBurnEvent` to the forge event bus.  This would be a compat layer unlike the former which are inherit Forge API features.

**Comment by DaddyP1G** [comment]
I noticed this was older, but working on my 1.19 + update for my modpack,  I'd love to include FD, but sorta stuck without this integration.
